src/components/Header.js

```diff
@@ -1,11 +1,41 @@
-import React from "react"
-
-const Header = () => {
-return (
-<>
-<h2>Header component</h2>
-</>
-)
+import React, { useState } from 'react';
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+} from 'reactstrap';
+
+function Header(props) {
+  const [collapsed, setCollapsed] = useState(true);
+
+  const toggleNavbar = () => setCollapsed(!collapsed);
+
+  return (
+    <div>
+      <Navbar color="info" light>
+        <NavbarBrand href="/" className="me-auto">
+          PixelPalace 
+        </NavbarBrand>
+        <NavbarToggler onClick={toggleNavbar} className="me-2" />
+        <Collapse isOpen={!collapsed} navbar>
+          <Nav navbar>
+            <NavItem>
+              <NavLink href="/components/ApartmentIndex/">See Your Apartments</NavLink>
```

Comment:
NavLinks to internal pages should not have an `href` 
This will prevent correct React functionality. This NavLink needs to come from React router.